### ISSUE TITLE
Is it possible to disable localstack services?

### ISSUE THREAD

**Issue by gpiotti** [commandeer/open#230]
If I only spin up certain services in localstack I keep getting error notifications from commandeer because it doesn't find it. For example I didn't enable Glue
"Glue crawlers refres failed - null"

If I look on localstack container logs it says
localstack.utils.server.http2_server.HTTPErrorResponse: no provider exists for service glue

I tried letting blank urls in AWS Endpoints but it didn't work

Thanks!

**Comment by bwship** [comment]
You mean, suppressing the error messages in the top nav for services that you aren't running on LocalStack?  If so, yea, we can do that.  We have a fix for the null part of the error message, but I think we can do something else for the alerts so that it doesn't get so annoying everytime you select an account.

**Comment by gpiotti** [comment]
yes, exactly that, or may be not even make the requests to localstack if the endpoints url are in None

**Comment by bwship** [comment]
[gpiotti](https://github.com/gpiotti) - verison 3.0.5 is coming out in the next hour.  These are now suppressed in that release.

**Comment by gpiotti** [comment]
thank you!
